client/src/components/SearchSitterForm/SearchSitterForm.tsx

```diff
@@ -0,0 +1,138 @@
+import { ChangeEvent, useState, SyntheticEvent } from 'react';
+import useStyles from './useStyles';
+import SearchIcon from '@material-ui/icons/Search';
+import DateRangeIcon from '@material-ui/icons/DateRange';
+import InputBase from '@material-ui/core/InputBase';
+import Autocomplete from '@material-ui/lab/Autocomplete';
+import { User } from '../../interface/User';
+import { DatePicker, MuiPickersUtilsProvider } from '@material-ui/pickers';
+import DateFnsUtils from '@date-io/date-fns';
+import { MaterialUiPickersDate } from '@material-ui/pickers/typings/date';
+import { IconButton } from '@material-ui/core';
+import { format } from 'date-fns';
+import isValid from 'date-fns/isValid';
+
+interface Props {
+  search: string;
+  handleChange: (event: ChangeEvent<HTMLInputElement>, newInputValue: string) => void;
+  selectedStartDate: Date | null;
+  handleStartDateChange: (date: MaterialUiPickersDate) => void;
+  selectedEndDate: Date | null;
+  handleEndDateChange: (date: MaterialUiPickersDate) => void;
+}
+
+const SearchSitterForm = ({
+  search,
+  handleChange,
+  selectedStartDate,
+  handleStartDateChange,
+  selectedEndDate,
+  handleEndDateChange,
+}: Props): JSX.Element => {
+  const [open, setOpen] = useState(false);
+  const [options, setOptions] = useState<User[]>([]);
+  const [loading, setLoading] = useState(false);
+
+  const classes = useStyles();
+
+  const formatWeekSelectLabel = (date: any, invalidLabel: string) => {
+    const dateClone = new Date(date);
+
+    return dateClone && isValid(dateClone) ? `${format(dateClone, 'd MMMM')}` : invalidLabel;
+  };
+
+  const renderDay = (date: any) => {
```

Comment:
For that one it shows as MaterialUiPickersDate for the type, but I can't cast that to a new object, I am not sure how what data type it actually is because casting to a new date actually works as expected